webapp/app.py

```diff
@@ -25,6 +26,17 @@ def index():
     return render_template("index.html")
 
 
+@app.route("/download/<osname>")
+def osredirect(osname):
+    if "macos" not in osname and "windows" not in osname:
+        return render_template("index.html")
```

Comment:
Wouldn't a redirect to "/#install" be better here?

Reply:
yes, I wasn't sure we wanted to redirect or just send them to a different route, but I don't know flask (or python) very well.